python/cuml/test/test_pickle.py

```diff
@@ -185,3 +190,19 @@ def test_neighbors_pickle(tmpdir, datatype, input_type, model, nrows,
 
     assert array_equal(D_before, D_after)
     assert array_equal(I_before, I_after)
+
+@pytest.mark.parametrize('datatype', [np.float32, np.float64])
+@pytest.mark.parametrize('input_type', ['dataframe', 'ndarray'])

```

Comment:
```suggestion

```

test_input_utils tests the input for all algorithms, no need to have different input_types anymore

Reply:
Okay! Should I input on cudf then, or numpy?

Reply:
I would do numpy